dockstore-webservice/src/main/java/io/dockstore/webservice/resources/DockerRepoResource.java

```diff
@@ -792,13 +792,20 @@ public List<Tool> allPublishedContainers(
         @ApiParam(value = "Sort column") @DefaultValue("stars") @QueryParam("sortCol") String sortCol,
```

Comment:
> This PR redirects 500 errors that occur when providing invalid filters for the endpoints /workflows/published and /containers/published.

Did some testing on QA to understand the problem and it appears that the 500s aren't from the invalid filters because `curl -X 'GET' "https://qa.dockstore.org/api/containers/published?offset=1&limit=42&filter=string%20value&sortCol=stars&sortOrder=string" -H "accept: application/json"` works. Tried each query parameter and it looks like the only problematic one is `sortCol` when you put any value that is not a field in `Entry`.

Untested, but I think the problematic line is [here](https://github.com/dockstore/dockstore/blob/afa15d20dbcc4b8479ac30d9bc8e9a219c4f37a1/dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/EntryDAO.java#L401) where it gets the `sortCol` from the entry. If it doesn't exist then I'm guessing it throws. The other query parameters in this function have safe defaults if the query parameter value doesn't make sense. For example, if the `sortOrder` isn't `desc` then it just defaults to `asc` [here](https://github.com/dockstore/dockstore/blob/afa15d20dbcc4b8479ac30d9bc8e9a219c4f37a1/dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/EntryDAO.java#L406) so you can't really have an invalid value. I think it's better to handle the error in this function instead of doing a `try/catch` in the Resource files because this fixes the root cause. Bonus: fixing it in that function fixes it for both tools/workflows endpoints.